src/ws-api/ws-api.ts

```diff
@@ -0,0 +1,64 @@
+import {
+    AnyFunction,
+    AnyFunctionWithThis,
+    OnMessageCb,
+    WebsocketClient,
+    WsProviderConnector
+} from '../connector/ws'
+import {ActiveOrdersWebSocketApi} from './active-websocket-orders-api'
+import {RpcWebsocketApi} from './rpc-websocket-api'
+import {WsApiConfigWithProvider} from './types'
+
+export class WebSocketApi {
+    public rpc: RpcWebsocketApi
+
+    public order: ActiveOrdersWebSocketApi
+
+    private ws: WsProviderConnector
+
+    constructor(config: WsApiConfigWithProvider) {
```

Comment:
1. Here pass only `provider`
2. Create `static new(configOrProvider: WsApiConfig | WsProviderConnector)` because we don't need config in case we've already provided provider